platforms/software/test-suites-base/src/main/java/org/gradle/testing/base/plugins/TestSuiteBasePlugin.java

```diff
@@ -29,8 +33,23 @@
  */
 @Incubating
 public abstract class TestSuiteBasePlugin implements Plugin<Project> {
+
     @Override
     public void apply(Project project) {
-        project.getExtensions().create(TestingExtension.class, "testing", DefaultTestingExtension.class);
+        TestingExtension testing = project.getExtensions().create(TestingExtension.class, "testing", DefaultTestingExtension.class);
+        project.getPlugins().apply(ReportingBasePlugin.class);
+        Provider<Directory> testReportsDir = project.getExtensions().getByType(ReportingExtension.class).getBaseDirectory().dir(TestingBasePlugin.TESTS_DIR_NAME);
+        testing.getResults().getHtmlReport().set(testReportsDir.map(it -> it.dir("aggregated-results")));
+
+        // All test suite targets get aggregated automatically
+        // TODO: Do we _always_ want to aggregate? How fast/slow is aggregation?
```

Comment:
💭 I think the `testing` extension should allow for opting-out of aggregation via a property.